src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const isMan = sex === 'm';
```

Comment:
You can shorten your jsx code further
```suggestion
  const partner = sex === 'm' ? 'wife' : 'husband';
```